src/libs/actions/Task.js

```diff
@@ -756,6 +751,18 @@ function isTaskAssigneeOrTaskOwner(taskReport, sessionAccountID) {
     return sessionAccountID === getTaskOwnerAccountID(taskReport) || sessionAccountID === getTaskAssigneeAccountID(taskReport);
 }
 
+/**
+ * Clears any possible stored errors for a specific field on a task report
```

Comment:
NAB: More simply; "Clear errors from editing a task".

Actually, we don't need a description at all. It's quite simple.

https://github.com/Expensify/App/blob/a30f5499f1703dfecaa6fd8258b3628c922ffb15/contributingGuides/STYLE.md#L170